packages/perseus/src/perseus-types.ts

```diff
@@ -1215,7 +1218,7 @@ export type PerseusNumberLineWidgetOptions = {
     // The correct relative value. default: "eq". options: "eq", "lt", "gt", "le", "ge"
     correctRel: string | null | undefined;
     // This is the correct answer. The answer is validated (as right or wrong) by using only the end position of the point and the relation (=, &lt;, &gt;, ≤, ≥).
-    correctX: number;
+    correctX: number | null;
```

Comment:
The measurer widget component handles null values.

Reply:
\<rant target='nobody-in-particular'>
I understand the difference between undefined and null, but so often we use these two values in the same way. IMHO it's a paper cut I wish we could remove eventually.
\</rant>